### PR TITLE
Added new simple scripts to facilitate header/space check

### PR DESCRIPTION
**Pull request by frheault**
With the new tools in Dipy being quite robust and seeing how complex it is for non-experts to play with header/space, these new simple scripts could help everyone.

First script can print the raw header from trk, tck, nii, nii.gz and mgz (pretty print of the dictionary)
And the second verify if all inputs have the same spatial attribute, support trk, nii, nii.gz and check if the affine, dimensions, voxel_size and voxel_order are the same.

**100th PR !**

### CODE REVIEW

File: scripts/scil_summarize_bundles_similarity_measures.py [hunk 1]
```diff
@@ -319,7 +319,7 @@ def main():
                     measure_dict[measure_name])
 
     with open(args.out_json, 'w') as outfile:
-        json.dump(output_measures_dict, outfile)
+        json.dump(output_measures_dict, outfile, indent=1)
```
Comment (ppoulin91) [inline scripts/scil_summarize_bundles_similarity_measures.py]: Sneaky Francois

File: scripts/scil_verify_space_attributes_compatibility.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    p = argparse.ArgumentParser(description=__doc__,
+                                formatter_class=argparse.RawTextHelpFormatter)
+
+    p.add_argument('in_files', nargs='+',
+                   help='List of file to compare (trk and nii).')
+
+    return p
+
+
+def main():
```
Comment (ppoulin91) [inline scripts/scil_verify_space_attributes_compatibility.py]: Maybe add a success message...?

File: scripts/scil_print_header.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    assert_inputs_exist(parser, args.in_file)
+
+    _, in_extension = split_name_with_nii(args.in_file)
+
+    if in_extension in ['.tck', '.trk']:
+        header = nib.streamlines.load(args.in_file, lazy_load=True).header
+    elif in_extension in ['.nii', '.nii.gz', '.mgz']:
+        header = dict(nib.load(args.in_file).header)
+    else:
+        parser.error('{} is not a supported extension.', in_extension)
```
Comment (AntoineTheb) [inline scripts/scil_print_header.py]: Missing `format` to string

File: scripts/scil_summarize_bundles_similarity_measures.py (hunk 1 above)
Comment (arnaudbore) [inline scripts/scil_summarize_bundles_similarity_measures.py]: Use add_json_args and use it here.

File: scripts/scil_print_header.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+    if args.keys:
+        for key in args.keys:
+            if key not in header:
+                parser.error('Key {} is not in the header of {}.'.format(key,
+                             args.in_file))
+            print(" '{}': {}".format(key, header[key]))
+    else:
+        pp = pprint.PrettyPrinter(indent=1)
+        pp.pprint(header)
```
Comment (arnaudbore) [inline scripts/scil_print_header.py]: Why use pprint and not json stuff here ?

File: scripts/scil_summarize_bundles_similarity_measures.py (hunk 1 above)
Comment (frheault) [reply scripts/scil_summarize_bundles_similarity_measures.py]: C'est actually un erreur, je vais l'enlever

File: scripts/scil_print_header.py (hunk 4 above)
Comment (EmmaRenauld) [reply scripts/scil_print_header.py]: I tried and it's actually a lot more complicated than the print.
The header is composed of bytes, not strings. I would have to loop on all keys and values to convert to strings.
I recommand to keep as is.

**Pull request merged**